index.js

```diff
@@ -114,7 +114,7 @@ var publicAgent = {
     return agent.addTransactionLabel(key, value);
   },
 
-  start: function(projectConfig) {
+  startAgent: function(projectConfig) {
```

Comment:
It seems odd for the constructor to return an object with a `startAgent` property that in turn produces an object with another `startAgent` property.